components/Backgrounds/Basic-BG.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function BasicBG() {
+  return <Background />;
+}
+
+const Background = styled.main`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: auto;
+  width: 100vw;
+  background: conic-gradient(
+    from 180deg at 50% 50%,
+    var(--color15) 0deg,
+    var(--color17) 50.63deg,
+    var(--color16) 166.87deg,
+    var(--color18) 241.88deg,
+    var(--color15) 360deg
+  );
+  z-index: -1;
+`;
```

Comment:
Done and done! I kept the name of the file as Basic-BG.js for the moment, because I would like to use different backgrounds for different stages of the game. Wouldn't do me much good if they all were named StyledMain...

Reply:
You could make it customisable! Styled Components can take props as well... but maybe that's something for the next refactoring round...